index.js

```diff
@@ -1,7 +1,21 @@
-var assign = require("object-assign");
-var stream = require("stream");
-var babel  = require("babel-core");
-var util   = require("util");
+var assign   = require("object-assign");
+var stream   = require("stream");
+var babel    = require("babel-core");
+var util     = require("util");
+
+var canCompile = require("babel-cli/lib/babel/util").isCompilableExtension;
+function arrayify(val, mapFn) {
+  if (!val) return [];
+  if (typeof val === "boolean") return arrayify([val], mapFn);
```

Comment:
Is there actually a time when someone would pass a boolean to browserify? The function was deleted because it's gross.